generator/syndesis-template.go

```diff
@@ -100,23 +105,29 @@ var syndesisContext = Context{
 // TODO: Update with product image references here
 var productContext = Context{
 	Images: images{
+		ImageStreamNamespace: "fuse-ignite",
+		SyndesisImagesPrefix: "syndesis",
```

Comment:
I guess this eventually will also change to `ignite`, right ?

Reply:
This kind of doesn't matter for product as we're only going to be used pre-populated image streams...